### PR TITLE
Add uCoz detection

### PR DESCRIPTION
**Pull request by AlexByte**
Sites to test:

- [1](http://torrent-games.net)
- [2](http://n-torrents.ru/)

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -8307,6 +8307,16 @@
 			"script": "use\\.typekit\\.com",
 			"website": "http://typekit.com"
 		},
+		"uCoz": {
+			"cats": [
+				1
+			],
+			"headers": {
+				"Set-Cookie": "uCoz="
```
Comment (jvoisin) [inline src/apps.json]: I'm a bit afraid that this will cause too much false-positives :/

Comment (AlexByte) [inline src/apps.json]: Unreasonable words.

Comment (jvoisin) [inline src/apps.json]: Not for javascript packers or base64 ;)

Comment (AlexByte) [inline src/apps.json]: What is the general relationship between the header and javascript?

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**